tests/tests_pytorch/test_cli.py

```diff
@@ -288,6 +290,21 @@ def test_lightning_env_parse(cleandir):
     assert cli.config.fit.trainer.logger is False
 
 
+def test_lightning_cli_and_trainer_kwargs_override(cleandir):
```

Comment:
Test doesn't pass on master, because the CLI passes all arguments to the Trainer, including the defaults, and so env variables would never take precedence.